scripts/audio.js

```diff
@@ -317,3 +330,42 @@ H5P.Audio.prototype.getCurrentState = function () {
     };
   }
 };
+
+/**
+ * @public
+ * Disable button.
+ * Not using disabled attribute to block button activation, because it will
+ * implicitly set tabindex = -1 and confuse ChromeVox navigation. Clicks handled
+ * using "pointer-events: none" in CSS, keydown on enter or space prevented by
+ * interception in event listener on button.
+ */
+H5P.Audio.prototype.disable = function () {
+  if (this.$audioButton) {
+    this.$audioButton.toggleClass(H5P.Audio.BUTTON_DISABLED, true);
+  }
+};
+
+/**
+ * @public
+ * Enable button.
+ */
+H5P.Audio.prototype.enable = function () {
+  if (this.$audioButton) {
+    this.$audioButton.toggleClass(H5P.Audio.BUTTON_DISABLED, false);
+  }
+};
+
+/**
+ * @public
+ * Check if button is enabled.
+ * @return {boolean} True, if button is enabled. Else false.
+ */
+H5P.Audio.prototype.isEnabled = function () {
```

Comment:
Same thing applies here for the naming. isEnabled is too generic.

Reply:
Changed